src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.Searcher.cs

```diff
@@ -121,33 +126,45 @@ private class Searcher
                 }
             }
 
-#pragma warning disable CS0618 // MatchKind is obsolete
             private void ReportMatchResult(Project project, INavigateToSearchResult result)
             {
+                var builder = ArrayBuilder<Span>.GetInstance();
+                foreach (var s in result.NameMatchSpans)
+                {
+                    builder.Add(new Span(s.Start, s.Length));
+                }
```

Comment:
nit: add an empty line after }

Reply:
nit: consider: using the .ToSpan extension method.